src/connectors/refinement-list/__tests__/connectRefinementList-test.ts

```diff
@@ -520,6 +520,35 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/refinement-
     ]);
   });
 
+  it('Provides search results within transformItems', () => {
+    const transformItems = jest.fn((items) => items);
+    const makeWidget = connectRefinementList(() => {});
+    const widget = makeWidget({
+      attribute: 'category',
+      transformItems,
+    });
+
+    const helper = jsHelper(createSearchClient(), '');
+
+    widget.init!(createInitOptions({ helper, state: helper.state }));
+    widget.render!(
+      createRenderOptions({
+        results: new SearchResults(helper.state, [
```

Comment:
if results is saved in a variable before rendering, it can checked in the called with

Reply:
(same for all other added tests)

Reply:
Done in 7f31fe1c.